packages/terra-visually-hidden-text/src/terra-dev-site/doc/example/NonNullVisuallyHiddenText.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import VisuallyHiddenText from 'terra-visually-hidden-text/lib/VisuallyHiddenText';
+
+export default () => (
+  <p>
+     Focus in this section to hear screen reader only text
+    <VisuallyHiddenText tabIndex="0" text="This is read by a screen reader" />
```

Comment:
I think verifying null & non-null should be under the test directory and a non-null example should be under the docs dir.